DataGenerator/src/main/java/org/opensearch/migrations/data/workloads/Geonames.java

```diff
@@ -9,6 +9,9 @@
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 
+import static org.opensearch.migrations.data.FieldBuilders.GEO_POINT;
+import static org.opensearch.migrations.data.FieldBuilders.INTEGER;
+import static org.opensearch.migrations.data.FieldBuilders.LONG;
```

Comment:
Let's continue to not allow static imports outside of test code (this is in src, not testFixture)
Adding the classname will give names like "INTEGER" some more context and clarity

Reply:
Thanks for the feedback, I don't agree with the style preference.  I would prefer to merge this change as is. 

I've created MIGRATIONS-2141 [1] to automate enforcement of style issues, lets use tooling to align our preferences.

- [1] https://opensearch.atlassian.net/browse/MIGRATIONS-2141